### PR TITLE
[zh-cn]: update the translation of error.prototype.toString() method

### PR DESCRIPTION
**Pull request by T34-active**

### CODE REVIEW

File: files/zh-cn/web/javascript/reference/global_objects/error/tostring/index.md [hunk 1]
```diff
@@ -84,4 +86,4 @@ console.log(e5.toString()); // 'hello'
 
 ## 参见
 
-- [`core-js`](https://github.com/zloirock/core-js) 中关于 [`Error.prototype.toString` 的 polyfill](https://github.com/zloirock/core-js#ecmascript-error)（修复了多个错误）
+- [在 `core-js` 中的 `Error.prototype.toString` 修复了许多错误](https://github.com/zloirock/core-js#ecmascript-error)
```
Comment (T34-active) [inline files/zh-cn/web/javascript/reference/global_objects/error/tostring/index.md:89]:
```suggestion
- [`core-js` 库中 `Error.prototype.toString` 方法的 Polyfill 修复的具体错误](https://github.com/zloirock/core-js#ecmascript-error)
```

File: files/zh-cn/web/javascript/reference/global_objects/error/tostring/index.md [hunk 2]
```diff
@@ -1,77 +1,79 @@
...
-
-  return name + ": " + msg;
+  return `${name}: ${msg}`;
 };
 ```
 
 ## 示例
 
-### 使用 toString()
+### 使用 toString() 方法
```
Comment (yin1999) [inline files/zh-cn/web/javascript/reference/global_objects/error/tostring/index.md:54]: Please be consistent with the English documentation. There is no disambiguation here and it's not necesssary to add the "method".

```suggestion
### 使用 toString()
```

File: files/zh-cn/web/javascript/reference/global_objects/error/tostring/index.md [hunk 3]
```diff
@@ -84,4 +86,4 @@ console.log(e5.toString()); // 'hello'
 
 ## 参见
 
-- [`core-js`](https://github.com/zloirock/core-js) 中关于 [`Error.prototype.toString` 的 polyfill](https://github.com/zloirock/core-js#ecmascript-error)（修复了多个错误）
+- [`core-js` 库中 `Error.prototype.toString` 方法的 Polyfill 修复的具体错误](https://github.com/zloirock/core-js#ecmascript-error)
```
Comment (yin1999) [inline files/zh-cn/web/javascript/reference/global_objects/error/tostring/index.md:89]: We use the fixed format:

```suggestion
- [`core-js` 中 `Error.prototype.toString` 的 polyfill（修复了多个错误）](https://github.com/zloirock/core-js#ecmascript-error)
```

**Review by yin1999 — APPROVED** [review]
Thank you @T34-active

**Pull request merged**